pkgs/by-name/ds/dscanner/package.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, lib, fetchFromGitHub, gnumake, ldc }:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dscanner";
+  version = "0.16.0-beta.4";
+
+  src = fetchFromGitHub {
+    owner = "dlang-community";
+    repo = "D-Scanner";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-5rct/HgQfmY79divQNxriKwSW+7N2poWHnV9NbDyJo8=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ gnumake ldc];
```

Comment:
stdenv includes make by default, you don't need this here. Also remove it from the function arguments above
```suggestion
  nativeBuildInputs = [ ldc ];
```